service/awsconfig/v10/key/key.go

```diff
@@ -230,6 +233,19 @@ func MasterImageID(customObject v1alpha1.AWSConfig) string {
 	return imageID
 }
 
+func MasterInstanceID(customObject v1alpha1.AWSConfig) string {
+	s := strconv.FormatInt(time.Now().UnixNano(), 10)
+
+	h := sha1.New()
+	h.Write([]byte(s))
+	timeHash := fmt.Sprintf("%x", h.Sum(nil))[0:5]
+
+	upperTimeHash := strings.ToUpper(timeHash)
+	upperClusterID := strings.ToUpper(ClusterID(customObject))
+
+	return fmt.Sprintf("MasterInstance%s%s", upperClusterID, upperTimeHash)
+}
```

Comment:
Just some magic to make the instance name random enough. Not going with the node IDs here since the concept of node IDs is rather vague and is expected to be improved/removed later.